src/datasources/db/v2/database-migrator.service.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { join } from 'path';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { Test, type TestingModule } from '@nestjs/testing';
+import { ConfigModule, ConfigService } from '@nestjs/config';
```

Comment:
We should us `@/config/configuration.module` instead of this directly.

Reply:
Unfortuantely, Since we need `env` variables, it's not possible to use `configuration.module`.